frontend/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,11 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ContentComponent } from './modules/workspace/content/content.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: '', redirectTo: '', pathMatch: 'full' },
+  {path:'content', component:ContentComponent}
```

Comment:
```suggestion
  { path: 'content', component: ContentComponent }
```
Until we configured Prettier, keep it clean and formatted